dts/vendor/nordic/nrf54h20.dtsi

```diff
@@ -920,7 +920,7 @@
 			};
 
 			nfct: nfct@985000 {
-				compatible = "nordic,nrf-nfct";
+				compatible = "nordic,nrf-nfct-v2";
```

Comment:
It would be better to do this in a separate commit, as a follow-up to 7249fac80d60b84d2ba9c2a1de6d42d8b3d9bd0c.

Reply:
split done.